### PR TITLE
PR[4:6] feat: integrate cobra with the --generate flag/command

### LINKED ISSUE #59: feat: integrate cobra with the --generate flag/command
This could also be a separate CLI tool TBD

### PR DESCRIPTION
**Pull request by zqueamish**
This integrates the generate command, to generate an HTML file. 
You use this by saying 
```
./Tech_Radar generate <path/to/merged_csvfile>
```

Resolves #59

### CODE REVIEW

**Comment by zqueamish** [comment]
The tests are currently failing, but we are adding the fix on another branch.

**Comment by Slug-Boi** [comment]
I would highly suggest that we wait with creating PRs on Novo main till we have confirmed that the branches actually work together on our dev main... Please ignore these for now and they may end up closed

**Comment by Slug-Boi** [comment]
> I would highly suggest that we wait with creating PRs on Novo main till we have confirmed that the branches actually work together on our dev main... Please ignore these for now and they may end up closed

We have agreed to convert this issue to a draft instead until it is ready for merging 👍

File: src/cmd/generate.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/*
+Copyright © 2024 NAME HERE <EMAIL ADDRESS>
+*/
```
Comment (Haruyzal) [inline src/cmd/generate.go:3]:
```suggestion
```

[the remark this answers is not in the document]
Comment (Slug-Boi) [reply src/cmd/generate.go:3]: This should indeed be removed nicely spotted

File: src/Merger/merger.go [hunk 2]
```diff
@@ -44,6 +46,35 @@ func readCsvContent(filepath string) ([]byte, error) {
...
 }
 
+func MergeFromFolder(folderPath string) error {
+	_, err := os.Stat(folderPath)
+	if os.IsNotExist(err) {
+		return errors.New("Folder does not exist or could not be found. \nError: " + err.Error())
+	} else if err != nil {
+		return err
+	}
+
```
Comment (fadur) [inline src/Merger/merger.go:56]: Avoid "else" statements at all costs😉
https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88

**Review by fadur — COMMENTED** [review]
Looks good. Looking forward to trying it out.

**Review by lasselundstenjensen — APPROVED** [review]

**Pull request merged**